catalogs/catalog-hive/src/test/java/com/datastrato/gravitino/catalog/hive/miniHMS/ScriptRunner.java

```diff
@@ -40,7 +40,10 @@ public class ScriptRunner {
   private final boolean stopOnError;
   private final boolean autoCommit;
 
+  @SuppressWarnings("DefaultCharset")
```

Comment:
Why do we ignore this?

Reply:
@qqqttt123 Of course we can fix it. But I'm not sure it's necessary. Here is the suggestion:
```
/home/mi/IdeaProjects/gravitino/catalogs/catalog-hive/src/test/java/com/datastrato/gravitino/catalog/hive/miniHMS/ScriptRunner.java:43: 警告: [DefaultCharset] Implicit use of the plaorm default charset, which can result in differing behaviour between JVM executions or incorrect behavior if the encoding of the data source doesn't match expectations.
  private final PrintWriter logWriter = new PrintWriter(System.out);
                                        ^
    (see https://errorprone.info/bugpattern/DefaultCharset)
  Did you mean 'private final PrintWriter logWriter = new PrintWriter(new BufferedWriter(new OutputStreamWriter(System.out, UTF_8)));' or 'private final PrintWriter logWriter = new PrintWriter(new BufferedWriter(new OutputStreamWriter(System.out, Charset.defaultCharset())));'?

```

What do you think? Should I fix it?

Reply:
We would better fix it.

Reply:
Done. I have fixed it.